nova/tests/unit/scheduler/filters/test_bigvm_filters.py

```diff
@@ -231,141 +232,110 @@ def call(a, b):
-                                  extra_specs={'host_fraction': 'any'},
+                                  extra_specs={'host_fraction': '-1,1.5'},
                                   name='random-name'))
         host = fakes.FakeHostState('host1', 'compute',
-                {'uuid': uuidsentinel.host1})
+                                   {'uuid': uuidsentinel.host1})
         self.assertFalse(self.filt_cls.host_passes(host, spec_obj))
 
     def test_extra_specs_full_positive(self):
         """test specified full size"""
-        CONF.set_override('bigvm_host_size_filter_uses_flavor_extra_specs',
-                          True, 'filter_scheduler')
-        CONF.set_override('bigvm_host_size_filter_host_fractions',
-                          {'full': '1', 'half': '0.5'}, 'filter_scheduler')
         spec_obj = objects.RequestSpec(
             flavor=objects.Flavor(memory_mb=CONF.bigvm_mb,
-                                  extra_specs={'host_fraction': 'full'},
+                                  extra_specs={'host_fraction': '1'},
                                   name='random-name'))
         host = fakes.FakeHostState('host1', 'compute',
-                {'uuid': uuidsentinel.host1})
+                                   {'uuid': uuidsentinel.host1})
         self.assertTrue(self.filt_cls.host_passes(host, spec_obj))
 
     def test_extra_specs_full_negative(self):
         """test specified full size"""
-        CONF.set_override('bigvm_host_size_filter_uses_flavor_extra_specs',
-                          True, 'filter_scheduler')
-        CONF.set_override('bigvm_host_size_filter_host_fractions',
-                          {'full': '1', 'half': '0.5'}, 'filter_scheduler')
         spec_obj = objects.RequestSpec(
             flavor=objects.Flavor(memory_mb=CONF.bigvm_mb,
-                                  extra_specs={'host_fraction': 'full'},
+                                  extra_specs={'host_fraction': '1'},
                                   name='random-name'))
         host = fakes.FakeHostState('host1', 'compute',
-                {'uuid': uuidsentinel.host1})
+                                   {'uuid': uuidsentinel.host1})
         self.filt_cls._HV_SIZE_CACHE[host.uuid] = CONF.bigvm_mb * 2 + 1024
         self.assertFalse(self.filt_cls.host_passes(host, spec_obj))
 
     def test_extra_specs_half_positive(self):
         """test specified half size"""
-        CONF.set_override('bigvm_host_size_filter_uses_flavor_extra_specs',
-                          True, 'filter_scheduler')
-        CONF.set_override('bigvm_host_size_filter_host_fractions',
-                          {'full': '1', 'half': '0.5'}, 'filter_scheduler')
         spec_obj = objects.RequestSpec(
             flavor=objects.Flavor(memory_mb=CONF.bigvm_mb,
-                                  extra_specs={'host_fraction': 'full,half'},
+                                  extra_specs={'host_fraction': '.5, 1'},
```

Comment:
Are both space and missing `0` here done on purpose? If yes, I'd rather test them in another test-function, that explicitly states that it's testing different `host_fraction` formats.

Reply:
Yes on purpose.
I will consolidate it (`'0.5,1'`), because I don't really expect `float()` to change behaviour anytime soon.